corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -554,6 +557,13 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
             // Reconcile state of "select all" checkbox
             self.ui.selectAllCheckbox.prop("checked", !_.difference(self._allCaseIds(), self.selectedCaseIds).length);
         },
+        verifySelectedCaseIdsLessThanMaxSelectValue: function () {
+            if (this.selectedCaseIds.length > this.maxSelectValue) {
+                let errorMessage = "You have selected more than the maximum selection limit of " + this.maxSelectValue +
+                ". Please uncheck some values to continue."
+                hqImport('hqwebapp/js/alert_user').alert_user(errorMessage, 'danger');
```

Comment:
[29e0779](https://github.com/dimagi/commcare-hq/pull/32416/commits/29e077971457868f357c013e0e3dbd98917ef1ab)

Reply:
Modified for hqRequire here: [ecc76fd](https://github.com/dimagi/commcare-hq/pull/32416/commits/ecc76fd1f82e04516c6e8e8b2cf5359e57567a2e) because the previous commit cause circular dependencies